frontend/public/components/operator-management.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import {Firehose, HorizontalNav, PageHeading} from './utils';
+import { referenceForModel } from '../module/k8s';
+import {SubscriptionsPage} from './operator-lifecycle-manager/subscription';
+import {PackageManifestsPage} from './operator-lifecycle-manager/package-manifest';
+import {InstallPlansPage} from './operator-lifecycle-manager/install-plan';
+import {
+  PackageManifestModel,
+  InstallPlanModel,
+  SubscriptionModel,
+} from '../models';
+
+const pages = [{
+  href: '',
+  name: 'Enabled Operators',
```

Comment:
@serenamarie125 @tlwu2013 What do we think about using "Available Operators" here to indicate that the Operators are available on-cluster to be ran after creating an OLM `Subscription`?

Reply:
Specifically, match the CTA from the Marketplace ("enable") with this nav item so you connect the two together.

Reply:
@alecmerdler per Sam, we can make small name changes later.  It would be great if we can get this through first, and then do a follow up ( for example maybe we go with Install versus Enable, but that will affect a few pieces of code).  Does that work for you ?